pyuvsim/analyticbeam.py

```diff
@@ -28,7 +28,7 @@ def peak_normalize(self):
 
     @profile
     def interp(self, az_array, za_array, freq_array):
-        # (Naxes_vec, Nspws, Nfeeds or Npols, freq_array.size, az_array.size)
+        # The azimuth here has the UVBeam convention: North of East(East=0, North=90)
```

Comment:
az_array = azimuth in degrees from East rotating towards the North. East=0, North = 90
za_array = zenith angle in degrees away from zenith. straight up=0, horizon=90
freq_array = frequencies in Hz

Three arrays may be in arbitrary shape, but shapes of all three must be the same.

Reply:
I added a full docstring.